crates/telio-proto/src/packet/control/derppoll.rs

```diff
@@ -42,11 +47,16 @@ pub struct DerpPollRequestMsg {
 
 impl DerpPollRequestMsg {
     /// Creates new request message from `peers`.
-    pub fn new(session: Session, peers: &[PublicKey]) -> Self {
+    pub fn new(session: Session, peers: &std::collections::HashSet<PublicKey>) -> Self {
         Self {
             session,
             msg: DerpPollRequest {
-                peers: RepeatedField::from_vec(peers.iter().map(|peer| peer.to_string()).collect()),
+                peers: RepeatedField::from_vec({
+                    let mut sorted_peers: Vec<String> =
+                        peers.iter().map(|peer| peer.to_string()).collect();
+                    sorted_peers.sort();
```

Comment:
`sort_unstable` is to be preferred unless the you need equal items to be in the same order as they initially were, because it's faster and uses less memory (probably not significant here, but still)

Reply:
Also, it looks like these fields weren't sorted before. Is there a reason they are sorted now? Or did I miss the sort somewhere?

Reply:
Since earlier there was a vector coming in from Config, and the test was based on vector that may have been incidentally sorted. There was some part where we do have sorted peers but maybe its not this one and sorting isn't really needed here at all!